src/document/DocumentAnnotator.ts

```diff
@@ -142,6 +149,20 @@ export default class DocumentAnnotator extends BaseAnnotator {
         this.getPages()
             .filter(({ dataset }) => dataset.loaded && dataset.pageNumber)
             .forEach(pageEl => this.renderPage(pageEl));
+
+        if (!this.annotatedEl) {
+            return;
+        }
+
+        if (this.mouseeventManager && !this.mouseeventManager.exists(this.annotatedEl)) {
```

Comment:
The usage of `exists` here is a bit confusing, as the manager only really acts upon the document element. Can we simplify by omitting the manager entirely?